db/migrate/20191122004321_remove_email_column_from_users_table.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEmailColumnFromUsersTable < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :users, :email, :string
```

Comment:
I wonder if you could'v gotten away with a migration here of 
`change_column(:users, :email, null: false, default: "")`
and removing the change to the email column in the devise migration.